alpha/lib/model/uiConf.php

```diff
@@ -813,4 +815,16 @@ public function setVersion($v, $subType = self::FILE_SYNC_UICONF_SUB_TYPE_CONFIG
 		
 		return parent::setVersion($v);
 	}
+
+	//v2 to v7
+	public function getV2Redirect()
+	{
+		return $this->getFromCustomData( self::CUSTOM_DATA_V2REDIRECT);
+	}
+
+	public function setV2Redirect($v)
+	{
+		KalturaLog::log(JSON_encode($v));
+		return $this->putInCustomData(self::CUSTOM_DATA_V2REDIRECT, $v);
```

Comment:
Why aren't you serializign the value before saving?

Reply:
is it needed? the value is saved and retrieved correct to/from DB